Titan/FodyWeavers.xml

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Weavers>
+  <Costura/>
```

Comment:
Maybe add the `NoCleanup` option for MacOS and Linux here? May be covered by change above if I read the docs correctly.

Reply:
Yea I dont think we have to change this if OS is not windows. The file is rendered useless anyways if Fody is not used at all.